package.json

```diff
@@ -1169,6 +1169,12 @@
             "default": null,
             "description": "Sets a path where MSBuild binary logs are written to when loading projects, to help diagnose loading errors."
           },
+          "dotnet.loadProjectsWithCSharpExtension": {
+            "scope": "machine-overridable",
```

Comment:
Unfortunately this doesn't remove it from user settings.  I saw no way to actually remove it from user settings :(.  Might be a limitation we have to live with.

Reply:
Hmm, does this work?

> resource - Resource settings, which apply to files and folders, and can be configured in all settings levels, even folder settings.

Reply:
Resource allows user settings as well.  However it seemed slightly more appropriate
<img width="691" alt="image" src="https://github.com/dotnet/vscode-csharp/assets/5749229/5924dd86-7a6e-4dfb-ae32-417b5b3ba4a1">